steps.md

```diff
@@ -294,12 +294,12 @@ bitrise-io/steps-gradle-runner:
 ##############################################
 
 * gradlew_path: $GRADLEW_PATH
-* gradle_task: assemble
   gradle_file: $GRADLE_BUILD_FILE_PATH
-▼ cache_level: only_deps [all only_deps none]
+* gradle_task: assemble
   gradle_options: --stacktrace --no-daemon
+▼ cache_level: only_deps [all only_deps none]
 
-[Filter Config]
+[Output Config]
```

Comment:
I suggest to keep with `Filter` rather than `Output`. You are right that with this fields you can control how the outputs will go however all of them are filter parameters.

Reply:
What about `Output Filters`? I really miss that this group is related to the step outputs...

Reply:
`Output Filters` would be the best I guess 👍